pkgs/run-sotest/src/run_sotest/__main__.py

```diff
@@ -46,6 +48,11 @@ def parse_cmdline_args():
         default=0,
         help="Priority to be assigned to a test run. Jobs with numerically higher priority preempt lower-priority jobs. Can be negative, default = 0.",
     )
+    parser.add_argument(
+        "--testrun_id_file",
```

Comment:
nit: I guess the :boat: has sailed on non-underscore arguments. This is really unconventional in Linux CLI tools.

Reply:
Hm, usually I'd say let's change it right now then but the rest of the script uses underscores, too, so changing it just for this one argument would be pretty confusing.

Thanks for the info though, I wasn't aware of that.